flutter.el

```diff
@@ -133,13 +133,13 @@ ARGS is a space-delimited string of CLI flags passed to
   "Helper function to initialize Flutter."
   (setq comint-process-echoes nil))
 
-
+
```

Comment:
Could you remove this extraneous space?

Reply:
This is not an extraneous space, I found that there are 3 `^L`, so I deleted it.
or did it set on purpose?

![image](https://user-images.githubusercontent.com/3764335/73252363-9b81f600-41f5-11ea-8143-8ee8b4ae40cd.png)

Reply:
It's there [on purpose](https://stackoverflow.com/a/1576895/448068).